src/commander.js

```diff
@@ -81,6 +82,11 @@ define([
             suffix: " ",
             tabSelectsMatch: true,
         });
+        atwho.autocomplete(
+            input,
+            {form: cmd.vellum.data.core.form, on: _.identity},  // fake mug
+            {useHashtags: true, tabSelectsMatch: true}          // options
```

Comment:
Similar to a comment I made on the overall PR, I think this atwho and the atwho in question properties should behave the same as much as possible, so I don't think this one should use `tabSelectsMatch`.